dotCMS/src/main/java/com/dotcms/graphql/datafetcher/ContentMapDataFetcher.java

```diff
@@ -0,0 +1,25 @@
+package com.dotcms.graphql.datafetcher;
+
+import com.dotcms.graphql.DotGraphQLContext;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.contentlet.transform.FolderToMapTransformer;
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.model.User;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.Map;
+
+public class ContentMapDataFetcher implements DataFetcher<Object> {
+    @Override
+    public Object get(final DataFetchingEnvironment environment) throws Exception {
+        try {
+            final User user = ((DotGraphQLContext) environment.getContext()).getUser();
```

Comment:
Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6090726)